web/core/modules/config/tests/src/Functional/ConfigEntityListTest.php

```diff
@@ -50,7 +50,7 @@ public function testList() {
 
     // Get a list of ConfigTest entities and confirm that it contains the
     // ConfigTest entity provided by the config_test module.
-    // @see config_test.dynamic.dotted.default.yml
+    // @see config_test.dynamic.dotted.services.yml
```

Comment:
Is this change intentional? It is part of Drupal core, but not core update took place.